src/ml_board/frontend/Dashboard/src/components/graphs/Graph.tsx

```diff
@@ -14,8 +17,8 @@ export default function Graph({ chart_id, exp_id, exp_data }: { chart_id: string
 
     const chartLabels = useAppSelector(state => selectChartLabelsById(state, chart_id));
     const experimentsDict = useAppSelector(state => selectExperimentsPerChartById(state, chart_id));
-
-    // prepare data (Warning Looping!)
+    const location = useLocation();
```

Comment:
is **useLocation** better or using the **idTab** in the **GlobalConfig** redux slice better? and isn't one of them is redundant then?